config/zowe-api-dev/alternative_start.sh

```diff
@@ -39,22 +39,24 @@ echo "active profile" $LOG_LEVEL
 echo "starting " ${DISCOVERY_CODE}
 
 _BPX_JOBNAME=${ZOWE_PREFIX}${DISCOVERY_CODE} java -Xms32m -Xmx256m -Xquickstart \
-    -Dibm.serversocket.recover=true \
+     -Dibm.serversocket.recover=true \
     -Dfile.encoding=UTF-8 \
     -Djava.io.tmpdir=/tmp \
-    -Dspring.profiles.active=https \
+    -Dspring.profiles.active=${APIML_SPRING_PROFILES:-https} \
     -Dspring.profiles.include=$LOG_LEVEL \
-    -Dserver.address=${ZOWE_IP_ADDRESS} \
+    -Dserver.address=0.0.0.0 \
     -Dapiml.discovery.userid=eureka \
     -Dapiml.discovery.password=password \
-    -Dapiml.discovery.allPeersUrls=https://${ZOWE_EXPLORER_HOST}:${DISCOVERY_PORT}/eureka/ \
+    -Dapiml.discovery.allPeersUrls=${ZWE_DISCOVERY_SERVICES_LIST} \
+    -Dapiml.logs.location=${WORKSPACE_DIR}/api-mediation/logs \
     -Dapiml.service.hostname=${ZOWE_EXPLORER_HOST} \
     -Dapiml.service.port=${DISCOVERY_PORT} \
     -Dapiml.service.ipAddress=${ZOWE_IP_ADDRESS} \
-    -Dapiml.service.preferIpAddress=false \
+    -Dapiml.service.preferIpAddress=${APIML_PREFER_IP_ADDRESS} \
```

Comment:
Instance.template.env:
Why reintroduce this?
<!--StartFragment--># this configuration is deprecated. Zowe start script will ignore this value and always set it to falseAPIML_PREFER_IP_ADDRESS=false<!--EndFragment-->

Reply:
we still have this one in start.sh, so we can remove it together in future